src/Language/PureScript/Docs/Convert.hs

```diff
@@ -173,8 +173,8 @@ convertDeclaration (P.TypeInstanceDeclaration _ constraints className tys _) tit
 
   childDecl = ChildDeclaration title Nothing Nothing (ChildInstance constraints classApp)
   classApp = foldl P.TypeApp (P.TypeConstructor className) tys
-convertDeclaration (P.FixityDeclaration fixity _ alias) title =
-  Just (Left ([title], AugmentFixity fixity alias))
+convertDeclaration (P.FixityDeclaration fixity _ Nothing) title =
+  Just (Left ([title], AugmentFixity fixity))
```

Comment:
I think we need a new case here, matching `(P.FixityDeclaration fixity op (Just name))`, returning `Just (Right (...))`. At the moment, as far as I can tell, operator alias declarations won't show up in the produced docs, because they aren't being matched here.